extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1146,6 +1146,46 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
       'image': true,
     },
   },
+
+  'ibeatanalytics': {
+    'requests': {
+      'host': 'https://ibeats.indiatimes.com',
+      'base': 'https://ibeats.indiatimes.com/iBeat/pageTrendlogAmp.html',
+      'pageview': '${base}?' +
+                '&h=ibeats.indiatimes.com' +
+                '&d=ibeats.indiatimes.com' +
+                '&url=${url}' +
```

Comment:
Thanks @1987yama3. I completely agree with your suggestion but we have different requirement on our site so we have to set most of the param as custom.
I have signed it with my personal gmail id,
Can i move this this project on official work mail id.
Please help me.